hw/xbox/dsp/dsp_dma.c

```diff
@@ -77,13 +109,21 @@ static void dsp_dma_run(DSPDMAState *s)
             s->eol = true;
         }
 
+        /* Decode control word */
+        bool unk0 = (control >> 0) & 1; /* FIXME: What does this do? */
```

Comment:
This has been documented at http://xboxdevwiki.net/DSP#Command_blocks now.

I want to rename this before merge.